### DISCUSSION TITLE
Is there a way to auto upload media from specific folder on pc to immich server?

### DISCUSSION THREAD

**Discussion by isgm570** [immich-app/immich#14260] [Q&A] [answered]
Hi everyone! I decided that i want to create a server for medeia (specifically for clips from games). At first i decided to create nextcloud server, it worked but didnt show thumbnails. So i decided to switch to immich, however i dont know if it has auto upload to the server

**Comment by schuhbacca** [comment, marked as answer]
There is no automatic upload. You would need to setup some cli job that runs periodically to upload the assets, or have the assets auto saved to like an SMB share that's configured as an external library.
